### DISCUSSION TITLE
Lodestar Standup 2024-35 (September 10, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#7068] [Standups]
### Meeting Info
[September 10, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/sep-10-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- v1.22.0 Planning
- Open Discussion
- Team Updates
- Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by amlevy327** [comment]
Just FYI: I came across [Stakers Union](https://www.stakersunion.com/) last week, a community for home-stakers with the mission of empowering individual stakers and preserving decentralization. 12 members took the time to complete the survey!

**Comment by philknows** [comment]
Would love to get a brainstorm going for ideas/topics to implement into our offsite schedule for planning.

**Comment by philknows** [comment]
Closing for #7087
